syft/frameworks/torch/mpc/falcon/falcon_tensor.py

```diff
@@ -0,0 +1,29 @@
+from syft.frameworks.torch.tensors.interpreters.replicated_shared import ReplicatedSharingTensor
+from operator import xor
+
+
+class FalconTensor(ReplicatedSharingTensor):
```

Comment:
I thought more on this and having a ```FalconTensor``` (having the suffix tensor on it) it means we can apply all operations from torch to this, but ```xor``` is not a method from a tensor.

My idea is to have the ```xor``` functionality in the ```FalconHelper``` (receives 2 RST and applies the xor).

Reply:
LE: Although, since we are writing a ```FalconTensor``` and xor was written under ```basic operations``` I think it might be a good idea.

Reply:
that's an excellent point!